definitions/ext-rf_scanner/definition.yml

```diff
@@ -21,14 +21,18 @@ synthesis:
   conditions:
     # The attribute’s value must match the provided value
     - attribute: eventType
+
       prefix: "THD_RF_TRANSACTION"
 
+
   # Telemetry attributes that should be extracted into entity tags.
   tags:
     ipAddr:
     osDetails:
+
+    serialNumber:
     distributionCenter:
-    inCradle:
+
       multiValue: false
```

Comment:
this multiValue configuration applied to the `inCradle` tag, you might want to remove it as well.